### PR TITLE
Mike/lofting enums

### PR DESCRIPTION
**Pull request by mlfarrell**

### CODE REVIEW

File: modeling-cmds/src/def_enum.rs [hunk 1]
```diff
@@ -136,6 +136,27 @@ define_modeling_cmd_enum! {
             pub tolerance: LengthUnit,
         }
 
+        /// Command for lofting sections to create a solid
+        #[derive(
+            Debug, Clone, Serialize, Deserialize, JsonSchema, ModelingCmdVariant,
+        )]
+        pub struct Loft {
+            /// The closed section curves to create a lofted solid from.
+            /// Currently, these must be Solid2D's
```
Comment (adamchalmers) [inline modeling-cmds/src/def_enum.rs:145]:
```suggestion
            /// Currently, these must be Solid2Ds
```

File: modeling-cmds/src/def_enum.rs [hunk 2]
```diff
@@ -136,6 +136,27 @@ define_modeling_cmd_enum! {
...
+            pub section_ids: Vec<Uuid>,
+            /// Degree of the interpolation.  Pass 2 for quadratic, and 3 for cubic interpolation in the V direction.
```
Comment (adamchalmers) [inline modeling-cmds/src/def_enum.rs:147]: Are these the only valid values? If so this should be an enum, like this [playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=62deb3ab15f40aa2b9c348c21f5c936b)

File: modeling-cmds/src/ok_response.rs [hunk 3]
```diff
@@ -75,6 +75,13 @@ define_ok_modeling_cmd_response_enum! {
             pub entity_ids: Vec<Uuid>,
         }
 
+        /// The response from the `Loft` command.
+        #[derive(Debug, Serialize, Deserialize, JsonSchema, ModelingCmdOutput)]
+        pub struct Loft {
+            ///The UUID of the newly created solid loft.
```
Comment (adamchalmers) [inline modeling-cmds/src/ok_response.rs:81]: This is a solid3d right?

**Review by adamchalmers — CHANGES_REQUESTED** [review]

File: modeling-cmds/src/def_enum.rs (hunk 2 above)
Comment (mlfarrell) [reply modeling-cmds/src/def_enum.rs:147]: No, any value is valid except I suppose 0.  So this should be kept as the integer I have

File: modeling-cmds/src/ok_response.rs (hunk 3 above)
Comment (mlfarrell) [reply modeling-cmds/src/ok_response.rs:81]: yes, but the reader should be able to infer that a solid loft is a solid 3d.  there's no such thing as a solid lofted 2D in our engine

**Review by adamchalmers — APPROVED** [review]

**Pull request merged**